devops/providers/azure-devops/templates/app_service_maven/tasks/service-inspect-repository.yml

```diff
@@ -0,0 +1,67 @@
+#####################
+# README: Outputs path of maven build directory and also outputs if folder structure of a 
+#   service is uniplatform (maven dir is in root) or multiplatform (maven dir is in core dir).
+#####################
+
+parameters:
+  stepName: ''
+
+steps:
+  - task: Bash@3
+    name: ${{ parameters.stepName }}
+    displayName: Determine source paths and build/test options
+    inputs:
+      targetType: 'inline'
+      script: |
+        #!/usr/bin/env bash
+        set -euo pipefail
+
+        # Emits an output variable that can be picked up by other jobs within this stage.
+        # Arguments:
+        #   $1: Variable Name
+        #   $2: Variable Value
+        # Capture the outputs using the following YAML syntax:
+        #   $[ dependencies.<ThisJobName>.outputs['<ThisStepName>.<VariableName>'] ]
+        function emit() {
+            # Echo value to log for diagnostics
+            echo "$1: $2"
+            # Echo to AzDO Pipeline (this line will be suppressed from execution logs)
+            echo "##vso[task.setvariable variable=$1;isOutput=true]$2"
+        }
+
+        function process_current_folder() {
+            # Search for a folder named <SERVICE_BASE>-core
```

Comment:
Should `SERVICE_BASE` be described? Is it, already? _(Ripping it out feels like something beyond the scope of this issue.)_

Reply:
Yeah, your comments reveal a couple of concerning things. I'm getting a strong feeling that we should spin-off more user stories for this. I'll come back to this comment shortly.

Reply:
This script along with two others (service-inspect-artifact, service-detect-cicd) are not actually being used. This was an over optimization from the first iterations for the **Shared Maven Service Pipeline**.  

These scripts were run and validated when iterating on them but never completely landed. What do you think about creating user story along the lines of reintegrating these scripts back in to eventually built more intelligence back into the **Shared Maven Service Pipeline**?.

Reply:
Another thing I noticed is that the call chain that's exercised when "core" is a component is completely an optional call chain. A maven service w/ a "core" component is not a required configuration. It's an option. I'll add some comments regarding the optional configurations and respond back here.

Reply:
Good sleuthing, @TechnicallyWilliams ... I think that the simpler the pipeline, the better. 
A good UI/UX designer I worked with used to look at his teams work and then say _"Subtract to release,"_ implying that, often, over complication is the enemy of utility.

Reply:
The explicit code path to support a maven service with a "core"/"common base" architecture has been removed.